src/wallet/getst/localStorageStore.d.ts

```diff
@@ -0,0 +1,14 @@
+export interface IStorageWrapper {
+  set(value: string | null | undefined): boolean;
+  get(): string | null | undefined;
+  delete(): boolean;
+}
+export declare class LocalStorageWrapper implements IStorageWrapper {
+  #private;
```

Comment:
is this a leftover?

Reply:
It is a prop from get-starknet but now mentioning it, it looks like I need to update that one.